src/MoveFilesToDocuments/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+project(MoveFilesToDocuments CXX)
```

Comment:
Let me know if you think I should use a different name.

Reply:
SGTM.